README.md

```diff
@@ -557,6 +558,66 @@ For detailed documentation on the InstructLab LLMs and their functions, see the
    ilab model chat --model ~/.cache/instructlab/models/mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf --temperature 0.5
    ```
 
+### 📇 Configure retrieval-augmented generation (developer preview)
+
+If you want to use the developer preview of retrieval-augmented generation (RAG), you need to enable developer preview features by setting `ILAB_FEATURE_SCOPE` to `DevPreviewNoUpgrade`. This tells InstructLab to enable developer preview features that might not be upgradable.  For many popular shells, this is done via the following:
+
+   ```shell
+   export ILAB_FEATURE_SCOPE=DevPreviewNoUpgrade
+   ```
+
+#### Converting documents
+
+Once you have enabled developer preview features, the first step is to convert your content into a structured form and then index that content for RAG to use with `ilab model chat`.  There are three options for conversion:
+
+*Converting documents from a directory:* If you have all the source documents (e.g., PDF, Markdown) that you want to convert in a single local directory, you can convert them:
+
+   ```shell
+      ilab rag convert --input-directory <path-to-document-directory> --output-directory <path-to-converted-document-directory> 
+   ```
```

Comment:
might be better for `<path-to-source-document-directory>`

Reply:
https://github.com/instructlab/instructlab/blob/8622643d309dfdd310d0497db1e0045da3944fca/src/instructlab/cli/rag/convert.py#L23
should be `--input-dir` and `--output-dir`

Reply:
These changes both look good.  I will adopt them.  Thx!